src/client/Microsoft.Identity.Client/Internal/Broker/NullBroker.cs

```diff
@@ -83,7 +77,8 @@ public Task<MsalTokenResponse> AcquireTokenByUsernamePasswordAsync(Authenticatio
 
         public Task<MsalTokenResponse> AcquireTokenByUsernamePasswordAsync(AuthenticationRequestParameters authenticationRequestParameters, AcquireTokenByUsernamePasswordParameters acquireTokenByUsernamePasswordParameters)
         {
-            throw new PlatformNotSupportedException();
+            _logger.Info("NullBroker - returning null on ROPC request.");
```

Comment:
Verbose?

Reply:
This is the pattern followed by other methods here. Do you think these should be changed to Verbose?